### PR TITLE
wip-0021: promote to `Proposed`

### PR DESCRIPTION
**Pull request by lrubiorod**

### CODE REVIEW

File: wip-0021.md [hunk 1]
```diff
@@ -170,6 +170,13 @@ The protocol improvements proposed herein will affect any library or client impl
 A reference implementation for the proposed protocol improvement can be found in the [pull request #2138](https://github.com/witnet/witnet-rust/pull/2138) of the [witnet-rust] repository.
 
 
+## Adoption Plan
+
+An activation date is proposed for March 1 2022 at 9am UTC, that is, protocol epoch #965760.
+
+From the perspective of TAPI, the signaling bit is `2` (`0b00000000000000000000000000000100`) and the `first_signaling_epoch` is set to #965760.
```
Comment (aesedepece) [inline wip-0021.md:177]: Shall we postpone this date? Or are we OK with leaving it as is and just assume that the first TAPI periods will complete without the proposal being passed?

Comment (drcpu-github) [reply wip-0021.md:177]: That seems rather silly?

**Comment by tmpolaczyk** [comment]
Can you promote WIP-0020 to proposed as well? Since they will probably be activated together.

File: wip-0021.md (hunk 1 above)
Comment (lrubiorod) [reply wip-0021.md:177]: jajajaja Those modifications are previous to Eth Denver conference. So, of course, date has to be updated

File: wip-0021.md [hunk 2]
```diff
@@ -170,6 +170,13 @@ The protocol improvements proposed herein will affect any library or client impl
 A reference implementation for the proposed protocol improvement can be found in the [pull request #2138](https://github.com/witnet/witnet-rust/pull/2138) of the [witnet-rust] repository.
 
 
+## Adoption Plan
+
+An activation date is proposed for April 5 2022 at 9am UTC, that is, protocol epoch #1032960.
+
+From the perspective of TAPI, the signaling bit is `2` (`0b00000000000000000000000000000100`) and the `first_signaling_epoch` is set to #1032960.
```
Comment (aesedepece) [inline wip-0021.md:177]: Please link TAPI to WIP-14

**Review by aesedepece — CHANGES_REQUESTED** [review]

**Review by tmpolaczyk — APPROVED** [review]

File: wip-0020.md [hunk 3]
```diff
@@ -130,11 +130,14 @@ A reference implementation for the proposed protocol improvement can be found in
 
 ## Adoption Plan
 
-An adoption plan will be proposed upon moving this document from the _Draft_ stage to the _Proposed_ stage.
+An activation date is proposed for April 5 2022 at 9am UTC, that is, protocol epoch #1032960.
+
+From the perspective of [TAPI], this proposal will be signaled along with the WIP-0021. This proposal adheres to the base implicit signal, and its first signaling epoch is also set to that of WIP-0021.
```
Comment (aesedepece) [inline wip-0020.md:135]: There's a contradiction here. It is either signaled along WIP-0021 (bit 2) or adheres to the base implicit signal (bit 0). You can simply copy and paste the same paragraph from WIP-0021 here.

**Review by aesedepece — CHANGES_REQUESTED** [review]

**Review by aesedepece — APPROVED** [review]
From the point of view of WIP-0001 and WIP-0014, these two WIPs satisfy all the formal requirements to be moved into the _Proposed_ stage.

**Pull request merged**